### PR TITLE
Polish near interaction docs

### PR DESCRIPTION
**Pull request by Troy-Ferrell**
## Overview
As part of the documentation review work, PR to polish near interaction page. 
- Re-organized content slightly
- Added media of profiles/inspectors/etc
- Added some additional code examples
- Added see also section
- Fixed markdown syntax errors
- Bolded/italicized/added URLs to some of existing content

## Changes
- Fixes: # .

### CODE REVIEW

File: Documentation/Input/HowToAddNearInteractivity.md [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
 
-Listen for touch and grab events by implementing [`IMixedRealityPointerHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler) and looking at the type of pointer that triggers your event. If the pointer is a PokePointer, the interaction is a touch. If the pointer is a SpherePointer, the interaction is a near grab. For touch interaction it is also possible to configure objects to raise touch events by implementing the [`IMixedRealityTouchHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityTouchHandler)
+There are three key steps to take in order to listen for touch and/or grab input events on a particular GameObject.
+
+1. Ensure the relevant pointer is registered in the main [MRTK Configuration Profile](../MixedRealityConfigurationGuide.md).
+1. Ensure the desired GameObject has the appropriate *NearInteraction** script component and [`Unity Collider`](https://docs.unity3d.com/ScriptReference/Collider.html).
+1. Implement the appropriate input handler interface on an attached script to the desired GameObject to listen for the touch/grab events.
```
Comment (wiwei) [inline Documentation/Input/HowToAddNearInteractivity.md]: Nit: What are your thoughts around clarifying "appropriate input handler"? As a reader if I'm reading this sequentially, I kinda see this and think "hmm I wonder what an appropriate input handler is?" - I think this confusion is cleared up once I keep reading, but it does give me slight pause and question (i.e. as I'm reading, I hit a location where I don't fully understand, should I go elsewhere and interrupt my reading flow to clarify what this is, or should I just keep going?). 

I don't know if there's any way to specify examples (or just something like the specific input handler is described in the Grab and Touch interaction sections below)

**Comment by wiwei** [comment]
From a readability perspective, lgtm. Not reviewed from a correctness perspective

File: Documentation/Input/HowToAddNearInteractivity.md (hunk 1 above)
Comment (Troy-Ferrell) [reply Documentation/Input/HowToAddNearInteractivity.md]: hmmm, probably worth while to list out all nearinteraction classes in a list and then put mention here of see sections below

Comment (Troy-Ferrell) [reply Documentation/Input/HowToAddNearInteractivity.md]: Actually just wrote in links to correct sections to simplify matter

File: Documentation/Input/HowToAddNearInteractivity.md [hunk 2]
```diff
@@ -29,55 +47,65 @@ public class PrintPointerEvents : MonoBehaviour, IMixedRealityPointerHandler
...
-## How to add near touch interactions to UnityUI
-1. Make sure you have a [PokePointer](Pointers.md#pokepointer) in your MRTK pointer profile. If you are using the default MRTK profile or the default HoloLens 2 profile, you will have this already. If you have a custom profile, open your custom profile and go to look under Input / Pointers  / Pointer Options. Make sure that for the "articulated hand" controller type you have an entry that points to the PokePointer prefab under MRTK.SDK/Features/UX/Prefabs, or create your own prefab and add a [PokePointer](Pointers.md#pokepointer).
+#### Touch code example
 
-1. Add a UnityUI canvas to your scene.
-
-1. Add a [`NearInteractionTouchableUnityUI`](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchableUnityUI) to your object.  
-
-1. Set "Events to Receive" to Pointer.
+The code below demonstrates a MonoBehavior that can be attached to a GameObject with a [`NearInteractionTouchable`](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchable) variant component and respond to touch input events.
```
Comment (julenka) [inline Documentation/Input/HowToAddNearInteractivity.md]: the touchable needs to be configured to receive touch events

**Pull request merged**